src/cli/main.c

```diff
@@ -46,7 +46,7 @@ main(int argc, char *argv[])
 	char		*provcode = NULL;
 	char		*network_name = NULL;
 	char		 new_name[64];
-	int		version; 
+	int		version = 0; 
```

Comment:
while we are here I guess that variable could be declared below the other flags, just after 'del_network = 0'

Reply:
kk lemme fix!